framework_tvb/tvb/adapters/visualizers/topographic.py

```diff
@@ -277,6 +277,11 @@ def launch(self, view_model):
         for i, array_data in enumerate(arrays):
             try:
                 data_array = TopographyCalculations.compute_topography_data(array_data, sensor_locations)
+                first_label = h5.load_from_index(connectivities_idx[0]).region_labels[0]
+                if first_label.lower().startswith('r'):
+                    data_array = numpy.rot90(data_array, k=1, axes=(0, 1))
+                else:
+                    data_array = numpy.rot90(data_array, k=-1, axes=(0, 1))
```

Comment:
The rotation seems to rely on the first connectivity only, but we could have different connectivities for the different arrays to display, isn't it?

Reply:
Hmm. Can  you please tell me how to generate an FCD with more connectivities? And then would it be enough just to add connectivities_idx[i] instead of connectivities_idx[0]?

Reply:
I think Lia refers to the viewer itself which is capable of displaying more than one ConnMeaures at once. Good point, I missed the connectivities_idx[0] there.

Reply:
Yes, I believe using connectivities_idx[i] would solve it.

To reproduce, you will need to have more than one Connectivity in your project, then produce ConnMeasures for both, and in the Topo viewer simply choose ConnMeasures in a mixed combination (linked with both the initial Connectivity dts).

Reply:
I tried a mixed combination, but the code says it is not possible and that we should choose a single connectivity for displaying. Please check lines 267-268 in topographic.py. Also, just removing that condition will produce further errors.

I suggest we leave this as is it now, maybe open a new task if we want to make the display work for different connectivities.

Reply:
Thanks for investigating. If only one conn can be chosen for all inputs, then it is ok for the code to remain unchanged. Pls just comment the code why conn _idx[0] is ok